src/main/java/reposense/wizard/OptionalPrompt.java

```diff
@@ -0,0 +1,35 @@
+package reposense.wizard;
+
+/**
+ * Represents an optional prompt that only calls the provided prompt if requested by the user.
+ */
+public class OptionalPrompt extends Prompt {
+    private final Prompt prompt;
+    private final static String YES_FLAG = "Y";
```

Comment:
Think there may be checkstyle errors here, static variables should always be above instance variables, and static should come before final

Reply:
Good catch, Ill fix this.